### DISCUSSION TITLE
Simple unit testing example

### DISCUSSION THREAD

**Discussion by midumitrescu** [pixijs/pixijs#10510] [Q&A]
Hi,

I was looking for some test examples with pixie and jest. 

Do you know of any good guide/example code? 

Thanks

**Comment by bigtimebuddy** [comment]
Many of our plugins use this package called `@pixi/extension-scripts` which has built-in support for running tests with Jest. You don't need to use extension-scripts as it's pretty specific toward developing pixi extensions. however, here is the relevant setup for how we use Jest:

https://github.com/pixijs/extension-scripts/blob/main/lib/configs/jest.mjs

Check out https://github.com/pixijs/gif for a running example.

Hope that helps.
